### PR TITLE
#3413 add name of other guardians to person

### PR DESCRIPTION
**Pull request by leventegal-she**
Closes https://github.com/hzi-braunschweig/SORMAS-Project/issues/3413

### CODE REVIEW

File: sormas-app/app/src/main/res/layout/fragment_person_edit_layout.xml [hunk 1]
```diff
@@ -471,13 +471,20 @@
...
                     android:id="@+id/person_fathersName"
-                    android:nextFocusForward="@+id/person_phone"
+                    android:nextFocusForward="@+id/person_nameOfOtherGuardians"
                     app:value="@={data.fathersName}"
                     app:imeOptions="actionNext"
                     style="@style/ControlSecondOfTwoColumnsStyle" />
 
             </LinearLayout>
 
+            <de.symeda.sormas.app.component.controls.ControlTextEditField
```
Comment (FredrikSchaeferVitagroup) [inline sormas-app/app/src/main/res/layout/fragment_person_edit_layout.xml:481]: As you probably know, this field also shows in for apps of non-german servers.

File: sormas-app/app/src/main/res/layout/fragment_person_read_layout.xml [hunk 2]
```diff
@@ -365,6 +365,11 @@
 
             </LinearLayout>
 
+            <de.symeda.sormas.app.component.controls.ControlTextReadField
```
Comment (FredrikSchaeferVitagroup) [inline sormas-app/app/src/main/res/layout/fragment_person_read_layout.xml:368]: As you probably know, this field also shows in for apps of non-german servers.

**Review by FredrikSchaeferVitagroup — CHANGES_REQUESTED** [review]
Should we add this field to the CaseExport and ContactExport?

**Comment by leventegal-she** [comment]
> Should we add this field to the CaseExport and ContactExport?

I don't think we need to export those fields because none of other similar fields like *Mothers's name, Father's name, etc.* are exported

File: sormas-app/app/src/main/res/layout/fragment_person_edit_layout.xml (hunk 1 above)
Comment (leventegal-she) [reply sormas-app/app/src/main/res/layout/fragment_person_edit_layout.xml:481]: Will be hidden after I merge the salutation field branch.

**Comment by FredrikSchaeferVitagroup** [comment]
> > Should we add this field to the CaseExport and ContactExport?
> 
> I don't think we need to export those fields because none of other similar fields like _Mothers's name, Father's name, etc._ are exported

Okay.

File: sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java [hunk 3]
```diff
@@ -428,6 +430,14 @@ public void setFathersName(String fathersName) {
 		this.fathersName = fathersName;
 	}
 
+	public String getNameOfOtherGuardians() {
```
Comment (FredrikSchaeferVitagroup) [inline sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java:433]: Also here, it should be namesOfOtherGuardians (plural), or?

**Review by FredrikSchaeferVitagroup — CHANGES_REQUESTED** [review]

**Review by FredrikSchaeferVitagroup — APPROVED** [review]

**Pull request merged**